lib/rules/no-absolute-internal-imports.js

```diff
@@ -0,0 +1,57 @@
+const relative = require('path').relative
+
+const message = 'No internal modules imports with absolute path'
+
+const getWithoutLastSegment = path => {
+    const parts = path.split('/').reverse()
```

Comment:
Was this tested under windows? Because paths are not normalized so on windows they may have `\` for folders instead of `/` but I'm not 100% sure of that. I just remember that sometime ago I had some struggles with working with paths on both unix and windows systems

Reply:
Is this a requirement?

Reply:
https://adverity.slack.com/archives/CK95U7PGC/p1583746023093900?thread_ts=1583745883.093800&cid=CK95U7PGC

Reply:
It's not a requirement. As clarified in the slack devteam channel: https://adverity.slack.com/archives/CK95U7PGC/p1583745883093800